RoutesAndTimetables/RoutesAndTimetables/RoutesAndTimetables/View/RouteDetailsPage.xaml

```diff
@@ -0,0 +1,97 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:ui="clr-namespace:RoutesAndTimetables.Ui"
+             x:Class="RoutesAndTimetables.Ui.View.RouteDetailsPage">
+    <ScrollView>
+        <AbsoluteLayout x:Name="absolutelayout">
+            <RelativeLayout x:Name="RootLayout">
```

Comment:
Is there a specific reason why you have this absolute layout with nothing but a relative layout inside? Couldn't you just remove the absolute layout?